cms/templates/widgets/header.html

```diff
@@ -230,9 +230,11 @@ <h2 class="sr-only">${_("Account Navigation")}</h2>
           <li class="nav-item nav-not-signedin-help">
             <a href="${get_online_help_info(online_help_token)['doc_url']}" title="${_('Contextual Online Help')}" target="_blank">${_("Help")}</a>
           </li>
-          <li class="nav-item nav-not-signedin-signup">
-            <a class="action action-signup" href="${reverse('signup')}">${_("Sign Up")}</a>
-          </li>
+          % if settings.FEATURES.get('ALLOW_PUBLIC_ACCOUNT_CREATION'):
```

Comment:
Use openedx.core.djangoapps.site_configuration.helpers.get_value when checking this flag, so that it can be overridden per-site. Default to the value in `settings.FEATURES`.